distributions/nuxt-ts/package.js

```diff
@@ -18,6 +20,17 @@ export default {
       ])
 
       await pkg.writePackage()
+
+      if (pkg.options.suffix && pkg.options.linkedDependencies) {
+        const tsconfig = readJSONSync(pkg.resolvePath('tsconfig.json'))
```

Comment:
Prefer async/await (non-Sync) methods when possible

Reply:
Oops 🙈, gonna change that thanks !